pandas/tests/series/test_arithmetic.py

```diff
@@ -64,6 +66,65 @@ def _constructor(self):
         result = op(m, 1)
         assert result.x == 42
 
+    def test_flex_add_scalar_fill_value(self):
+        # GH12723
+        s = Series([0, 1, np.nan, 3, 4, 5])
+
+        exp = s.fillna(0).add(2)
+        res = s.add(2, fill_value=0)
+        tm.assert_series_equal(res, exp)
+
+    pairings = [(Series.div, operator.truediv, 1), (Series.rdiv, ops.rtruediv, 1)]
```

Comment:
can you paramaterize

Reply:
this is constructing the parameters that go into parametrize on L88